webrtc.html

```diff
@@ -7141,9 +7141,11 @@ <h2>Dictionary RTCRtpContributingSource Members</h2>
               in [[!RFC6465]] if the RFC 6465 header extension is present,
               otherwise this member MUST be absent.</p>
               <p>For SSRCs, this MUST be converted from the level value defined
-              in [[!RFC6464]] if the RFC 6464 header extension is present,
-              otherwise the user agent must compute the value from the audio
-              data (the member must never be absent for audio receivers).</p>
+              in [[!RFC6464]]. This extension MUST be sent by user agents, and
+              as such should be present. But if the RFC 6464 header extension is
+              not present in the received packets (such as if the other endpoint
+              is not a user agent or is a legacy endpoint), this value SHOULD be
+              absent.</p>
```

Comment:
Keep it simple!

Reply:
SHOULD NOT be present.

Reply:
Recommend deleting "This extension MUST be sent by user agents, and as such should be present. But "